opg-mongodb-1604/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM registry.service.opg.digital/opg-base-1604
+
+ENV MONGODB_VERSION 3.0.15
+
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10 && \
```

Comment:
would it not be better to use the curl method like in kibana to fetch the key for apt so we avoid the intermittent keyserver errors